### ISSUE TITLE
implement a fine(r) grained handler system

### ISSUE THREAD

**Issue by zerwes** [Rosa-Luxemburgstiftung-Berlin/ansible-opnsense#12]
Currently the [RLS `main.yml`](https://github.com/Rosa-Luxemburgstiftung-Berlin/ansible-opnsense/blob/master/tasks/main.yml#L254-L275) tasks just has a split up of the apply handler into 2 steps:
 * `configctl filter sync` for a simpler and non-invasive sync of the filter rules
 * `configctl service reload all` for a hard reload of all services (resulting sometimes in interupted network traffic / restet of connections)
It would be desirable to have a finer control over the required steps for applying the changes. i.e. just a simple  change in the wireguard configuration should not require the restart of __all__ services.
So each task should notify just the required services for a restart.

**Comment by zerwes** [comment]
note to myself
`configctl service list`
https://docs.opnsense.org/development/backend/configd.html
```
# ls -1 /usr/local/opnsense/service/conf/actions.d/
actions_auth.conf
actions_captiveportal.conf
actions_configd.conf
actions_cron.conf
actions_dhcpd.conf
actions_dmidecode.conf
actions_dns.conf
actions_extended-dnsbl.conf
actions_filter.conf
actions_firmware.conf
actions_ftpproxy.conf
actions_health.conf
actions_hwprobe.conf
actions_ids.conf
actions_interface.conf
actions_ipfw.conf
actions_ipsec.conf
actions_lldpd.conf
actions_monit.conf
actions_netflow.conf
actions_netsnmp.conf
actions_openssh.conf
actions_openvpn.conf
actions_opnbe-core.conf
actions_opncentral-core.conf
actions_pfplugin.conf
actions_proxy.conf
actions_syslog.conf
actions_system.conf
actions_template.conf
actions_unbound.conf
actions_webgui.conf
actions_wireguard.conf
actions_zfs.conf
```

**Comment by zerwes** [comment]
src: `src/opnsense/service/conf/actions.d/*`

**Comment by zerwes** [comment]
time to start this: see #51 ...

**Comment by zerwes** [comment]
first steps done in #53 while implementing handler for alias

**Comment by zerwes** [comment]
#58 works for me ...
just using ansible-playbook -t dhcp,copy ... will bypass the big reload and leave you with the right dhcp config in /var/dhcpd//etc/dhcpd.conf and a reloaded service ...
can you approve this?

**Comment by zerwes** [comment]
for openvpn instances fixed in #63

**Comment by zerwes** [comment]
WIP ... still some handler to implement here, but I will close the main issue here, as the main stuff is implemented. Missing handlers should be covered in seperate issues (one per handler/topic)
